srcpkgs/libunwind/template

```diff
@@ -14,6 +14,12 @@ checksum=90337653d92d4a13de590781371c604f9031cdb50520366aa1e3a91e1efb1017
 
 CFLAGS="-fcommon"
 
+# Enable C++ exception handling (doesn't work too well on x86)
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*|i686*) ;;
+	*) configure_args+=" --enable-cxx-exceptions" ;;
+esac
```

Comment:
You're adding new functionality to the library, so update the entry in `common/shlibs` to `libunwind-1.5.0_3`